components/NavLayout/SideBar.js

```diff
@@ -68,13 +70,9 @@ export default function SideBar({ children, ...props }) {
             ClassMates
           </Typography>
           <Container className={styles.profileIcon} sx={{ flexGrow: 0 }}>
+            <Leaderboard />
             <Typography className={styles.profileName}>Current User</Typography>
-            <Avatar
-              className={styles.profileAvatar}
-              alt="Remy Sharp"
-              src={defaultProfilePic}
-              sx={{ width: 50, height: 50 }}
-            />
+            <ProfileMenu />
```

Comment:
Nice job on taking this out and creating new component to keep code concise and more readable